packages/mercury/docs/installation-for-angular.md

```diff
@@ -0,0 +1,372 @@
+
+In the following sections we provide examples of how to create those CSS bundles.
+
+#### 2.2.1. Using the CLI to create the CSS bundles
+
+> [!IMPORTANT]  
+> This is the way we recommend to create the bundles.
+
+Mercury exposes a CLI to automate the creation of bundles. This CLI comes with the installation of the `@genexus/mercury` dependency.
+
+Usage:
+
+```bash
+# Command line
+npx mercury <flags>
+
+# package.json script
+"build.scss": "mercury <flags>"
+```
+
+Consult the [CLI flags](#4-cli-flags) section to read the complete documentation for the CLI.
+
+Using the CLI:
+
+1. Add a `"build.scss"` script in your package.json and include the paths where the icons and fonts will be copied, also include the `outDir` path where the CSS will be generated.
+
+   For example:
+
+   ```json
+   "build.scss": "mercury --i={{ Icons final path }} --f={{ Fonts path final }} --outDir={{ CSS outDir path }}"
+   ```
+
+2. Use the `"build.scss"` script before building your application (dev server and prod build).
+
+   For example:
+
+   ```json
+   "dev": "npm run build.scss && ...",
+   "start": "npm run build.scss && ...",
+   "build": "npm run build.scss && ..."
+   ```
+
+3. Before using any Mercury or [Chameleon](https://github.com/genexuslabs/chameleon-controls-library) utility, you must register the bundle mapping. This mapping allows Mercury to map the bundle names with the real name of the CSS file (for example, `bundleToHashMappings["components/button"] --> components/button-9f82641938b85445`).
+
+   **IMPORTANT**: Run this JavaScript before using any Mercury or [Chameleon](https://github.com/genexuslabs/chameleon-controls-library) utilities.
+
+   ```ts
+   import { setBundleMapping } from "@genexus/mercury/dist/bundles";
+   import { bundleToHashMappings } from "{{ CSS outDir path }}/bundle-to-hash-mappings";
+
+   setBundleMapping(bundleToHashMappings);
+
+   // Other Mercury and Chameleon utilities...
+   ```
+
+#### 2.2.2. Using SASS to transpile the CSS bundles
+
+> [!WARNING]
+> We don't recommend this way, because the CLI does this under the hood in a much better and faster way. Also, this way does not hash the generated CSS, which leads to cache issues when re-deploying your application after updating your version of Mercury.
+
+1.  Install `sass` dependency to transpile the bundles.
+
+    ```bash
+    npm i -D sass
+    ```
+
+2.  Add a config file (called `config.scss`) in your project to determine the path to the assets.  
+    Include the following configuration:
+
+    ```scss
+    $font-face-path: "{{ Fonts final path }}";
+    $icons-path: "{{ Icons final path }}";
+    $globant-colors: false;
+    ```
+
+3.  Run the following command to transpile the bundles with the new path for the assets:
+
+    ```bash
+    npx sass --load-path=<path to config.scss directory> --no-source-map --style compressed <path to node_modules>/@genexus/mercury/dist/bundles/scss:{{ CSS outDir path }}
+    ```
```

Comment:
This option is no longer supported since the `0.21.0` version
https://github.com/genexuslabs/design-systems/releases/tag/v0.0.194


```suggestion
```